packages/expo/package.json

```
@@ -136,6 +136,7 @@
     "react-native-view-shot": "2.5.0",
     "react-native-webview": "5.4.6",
     "serialize-error": "^2.1.0",
+    "stacktrace-parser": "^0.1.5",
```

```suggestion
    "stacktrace-parser": "0.1.4",
```
Lock it, 0.1.5 is busted on iOS.